pyproject.toml

```diff
@@ -1,21 +1,25 @@
-[build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
+[tool.poetry]
+name = "aiproteomics"
+version = "0.1.0"
+description = ""
+authors = ["Robin Richardson <r.richardson@esciencecenter.nl>"]
```

Comment:
Is it possible to add more than one author here? You should obviously add yourself too

Reply:
done. anyone else?